### ISSUE TITLE
ForbiddenFunctions rule detection problem

### ISSUE THREAD

**Issue by icedevelopment** [friendsoftwig/twigcs#174]
The ForbiddenFunctions rule added in #67 doesn't detect function calls with spaces before the first parenthesis.

Example:
```twig
{% dump (foo) %}
{{ dump (foo) }}
```
won't be detected but

```twig
{% dump(foo) %}
{{ dump(foo) }}
```
will be.

@Ciloe any chance you could slightly modify the parser to handle these cases?

**Comment by Ciloe** [comment]
Yes, I will see that. Thanks for your feedback !

**Comment by icedevelopment** [comment]
As a side note `{% dump %}` isn't detected either.

**Comment by Ciloe** [comment]
Yes, but this is not a real function which is forbidden.

**Comment by icedevelopment** [comment]
True, technically its a tag in this case but I think it would be a very useful thing if this rule would detect it as well.
According to the comments in Symfony\Bridge\Twig\TokenParser\DumpTokenParser the syntax should be:
```
{% dump %}
{% dump foo %}
{% dump foo, bar %}
```
but it looks like `{% dump (foo) %}` is syntactically valid. On the other hand `{% dump (foo, bar) %}` will throw an exception.
